src/FlagProvider.test.tsx

```diff
@@ -148,7 +148,7 @@ test('A consumer should be able to get a variant when the client is passed into
     'consuming value isEnabled true'
   );
   expect(screen.getByText(/consuming value updateContext/)).toHaveTextContent(
-    'consuming value updateContext [object Promise]'
+    'consuming value updateContext undefined'
```

Comment:
Now the promise is consumed and because setting the context returns no value, this seems correct to me